modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/nodecollection/v0.1.1.json

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/nodecollection/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:NodeCollectionShape",
+      "@type": "sh:NodeShape",
+      "label": "elements of the collection defined like this: ",
+      "targetClass": "nsg:NodeCollection",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:memodelRelease",
+              "name": "MEModel release",
+              "description": "MEModel release.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/memodelrelease/v0.1.1/shapes/MEModelReleaseShape",
+              "class": "nsg:MEModelRelease",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:cellProperties",
+              "name": "Properties",
+              "description": "Location of circuit file containing general circuit geometry: cell placement, orientation, mvd3.",
```

Comment:
circuit cell properties: cell orientation, position, electrical types, morphological types...

Reply:
I'm not in favor of "nsg:cellProperties" as it doesn't convey the list you provided. A cell can have many other properties I guess. Can it be nsg:CircuitCellProperties ?

Reply:
ok

Reply:
Great.